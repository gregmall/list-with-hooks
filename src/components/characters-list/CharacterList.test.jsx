import React from 'react';
import { render } from '@testing-library/react';
import CharacterList from './CharactersList';

describe('List Component', () => {
  const characters = [{ name: 'rick', image: 'www.url.com' }, { name: 'morty', image: 'www.page.com' }];

  it('renders List component', () => {
    const { asFragment } = render(<CharacterList characters={characters} />);

    expect(asFragment()).toMatchSnapshot();
  });
});
