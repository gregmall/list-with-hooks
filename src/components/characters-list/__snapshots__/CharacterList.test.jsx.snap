// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List Component renders List component 1`] = `
<DocumentFragment>
  <ul
    data-testid="characters"
  >
    <li>
      <a
        href="/detail/undefined"
      >
        <div>
          <p>
            rick
          </p>
          <img
            src="www.url.com"
          />
        </div>
      </a>
    </li>
    <li>
      <a
        href="/detail/undefined"
      >
        <div>
          <p>
            morty
          </p>
          <img
            src="www.page.com"
          />
        </div>
      </a>
    </li>
  </ul>
</DocumentFragment>
`;
